source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -155,7 +155,7 @@ namespace PlayFab
         using clock = std::chrono::steady_clock;
         using Result = PlayFabEventBuffer::EventConsumingResult;
         std::shared_ptr<const IPlayFabEmitEventRequest> request;
-        size_t batchCounter = 0; // used to track uniqueness of batches in the map
+        size_t batchCounter = 1; // used to track uniqueness of batches in the map
```

Comment:
interesting to start from expecting a single batch to start. i think it does make sense, but it's a bit of a change to go from nothing (0 when we initialize) to something (well we will probably put something on there anyway).

Reply:
The reason for this change was actually for a different reason:
This is cast to a void*, where the value of 0 implies "nullptr" which was confusing me as I was debugging.
There's no harm in it being 1, and it was easier for me to interpret what customData was while debugging.

---
In reply to: [355049528](https://github.com/PlayFab/XPlatCoreTemplate/pull/60#discussion_r355049528) [](ancestors = 355049528)